src/internal/connector/exchange/service_restore.go

```diff
@@ -94,16 +88,12 @@ func RestoreExchangeContact(
 	return info, nil
 }
 
-// RestoreExchangeEvent restores a contact to the @bits byte
-// representation of M365 event object.
-// @param destination is the M365 ID representing Calendar that will receive the event.
-// Returns an error if input byte array doesn't parse into models.Eventable object
-// or if an error occurs during sending data to M365 account.
-// Post details: https://docs.microsoft.com/en-us/graph/api/user-post-events?view=graph-rest-1.0&tabs=http
-func RestoreExchangeEvent(
+// RestoreEvent wraps api.Events().PostItem()
+func RestoreEvent(
 	ctx context.Context,
 	bits []byte,
-	service graph.Servicer,
+	cli itemPoster[models.Eventable],
```

Comment:
nit: Why `cli` and not `itp` or similar?

Reply:
this should be `ip` now.